scripts/test.sh

```diff
@@ -53,7 +53,15 @@ function run_test() {
 
     # replace the dummy config (CONFIG_FILE_NAME) with the one used by this test
     /bin/cp "$config_file_path" "$CONFIG_FILE_NAME"
-    # in the future - validate config file here
+    # Validate config file
+    if npx hardhat --typecheck --config "$CONFIG_FILE_NAME" 1>/dev/null; then
+        echo "Config file is valid"
+    else
+        echo "Invalid config"
+        # Clean up the temporary config file
+        rm -f "$CONFIG_FILE_NAME"
+        exit 1
```

Comment:
exit or return? Won't exit cause the whole script to stop, but we want to run all the test?

Reply:
[c2077b6](https://github.com/0xSpaceShard/starknet-hardhat-plugin/pull/401/commits/c2077b6ebd6df6dd642cde161b03518fe5dd8246)